main/static/main/search-nav.js

```diff
@@ -4,6 +4,14 @@ let category;
 let search_type;
 let search_bar;
 
+var h5 = $('.card-body h5');
+var ks = $('.card-title').height();
+while ($(h5).outerHeight() > ks) {
+  $(h5).text(function(index, text) {
+    return text.replace(/\W*\s(\S)*$/, '...');
+  });
+}
+
```

Comment:
shouldn't this be in the tile styles page? And I think we can use const instear ov var?